src/foremast/runner.py

```diff
@@ -170,6 +171,8 @@ def create_dns(self):
             dnsobj.create_failover_dns(primary_region=primary_region)
         else:
             dnsobj.create_elb_dns(regionspecific=False)
+            if regionspecific_dns: #If true, Also create a region specific DNS record
+                dnsobj.create_elb_dns(regionspecific=True)
```

Comment:
Ah makes sense for now. Both records point to the ELB by default. Sounds fine by me for now.